src/stim/simulators/measurements_to_detection_events.test.cc

```diff
@@ -375,7 +375,7 @@ TEST(measurements_to_detection_events, file_01_to_dets_no_obs) {
     rewind(in);
     FILE *out = tmpfile();
 
-    stream_measurements_to_detection_events(
+    stream_measurements_to_detection_events<MAX_BITWORD_WIDTH>(
```

Comment:
`TEST_EACH_WORD_SIZE_W`

Ultimately there are gonna be very few uses of `MAX_BITWORD_WIDTH` anywhere in the code base, other than the python bindings.